app/src/test/java/com/example/workoutapp/WorkoutUnitTest.kt

```diff
@@ -34,7 +30,7 @@ import java.util.concurrent.TimeUnit
 class WorkoutUnitTest {
 
     lateinit var gam: GroupsActivity
-    lateinit var workoutService: WorkoutService()
+    lateinit var workoutService: WorkoutService
```

Comment:
Correct.  This should either be:
= WorkoutService()  (constructor call) or
: WorkoutService  (variable of type WorkoutService)

Since this is preceded with the lateinit modifier, it indicates that this should be a variable of type WorkoutService, and the code reviewer made the correct change.

To the group: I highly doubt this would have compiled as-is.  Except for TDD unit tests, which should be in their own branch, code you commit to Version Control should **always** compile.  I recommend you stop everything you are doing, and make sure your project builds, before proceeding on anything else.  Otherwise, you're building a house on an instable foundation.